content/en-us/reference/engine/classes/UnreliableRemoteEvent.yaml

```diff
@@ -48,14 +119,32 @@ methods:
     security: None
     thread_safety: Unsafe
   - name: UnreliableRemoteEvent:FireServer
-    summary: ''
-    description: ''
+    summary: |
+      Fires the `Class.UnreliableRemoteEvent.OnServerEvent|OnServerEvent` event on the
+      server from one client connected to the same `Class.UnreliableRemoteEvent`. Has a 900 byte limit to the payload of the event, otherwise event is dropped.
```

Comment:
```suggestion
      Fires the `Class.UnreliableRemoteEvent.OnServerEvent|OnServerEvent` event on the
      server from one client connected to the same `Class.UnreliableRemoteEvent`. Has a
      900 byte limit to the payload of the event, otherwise event is dropped.
```